[AI] Fixes and updates for skills (#191)
open-s4c/csb: .agents/skills/csb-analysis/template-csb-analysis-report.md

## Kernel Change Artifacts
- `<function>`: `[kernel changes](<benchmark>-<function>-kernel-changes.md)`

## Kernel Commit Analysis

> List commits in the Linux kernel, between Linux release (e.g. 6.6) used for benchmarking and the latest release, that affect the hot code paths of the benchmark run. The goal here is find commits that, if not already present, would be worth backporting.

Reviewer: ```suggestion
List commits in the Linux kernel, between Linux release (e.g. 6.6) used for benchmarking and the latest release, which affect the hot code paths of the benchmark run. The goal here is find commits that, if not already present, would yield better performance if backported.
```
